src/backend/access/spgist/spgscan.c

```diff
@@ -577,6 +577,12 @@ spggetbitmap(PG_FUNCTION_ARGS)
 	TIDBitmap  *tbm = (TIDBitmap *) PG_GETARG_POINTER(1);
 	SpGistScanOpaque so = (SpGistScanOpaque) scan->opaque;
 
+	if (tbm == NULL)
+		/* XXX should we use less than work_mem for this? */
+		tbm = tbm_create(work_mem * 1024L);
+	else if (!IsA(tbm, TIDBitmap))
+		elog(ERROR, "non hash bitmap");
+
```

Comment:
Is this due to merge error?

Reply:
It's GPDB modified. I should add a comment here.
I will add a comment for all xxgetbitmap functions.